### PR TITLE
Refactored isElementInViewport

### PR DESCRIPTION
**Pull request by itdrake91**
Refactored this function based on my recent experience with a certain site that had a lot of variety in the element heights (especially when things started stacking on mobile). Despite using a consistent value for `elementDivisorSize` like `0.1`, there were still a lot of instances where it didn't feel right. Essentially there's a big difference between 10% of an element that's 50px tall and 10% of an element that's 1200px tall. Now the threshold argument is based on the viewport height, not the element height, which seems to feel more natural/consistent. See https://jsfiddle.net/c9j9kahw/121/ for a test.

### CODE REVIEW

File: src/functions/scroll.ts [hunk 1]
```diff
@@ -6,61 +6,48 @@
...
  * fully visible in the viewport.
- * @param {object} config
- * @param {Element} config.element HTML Element node to target.
- * @param {number} [config.elementDivisorSize] Size of division of
- * element's height to offset. E.g. 2 is half the height, 3
- * is one-third the height, etc.
- * @param {boolean} [config.useBottomOffset] Determines if offset
- * generated from elementDivisorSize should be applied to
- * the bottom of the element.
+ * @param {Element} element HTML Element node to target.
```
Comment (geoffdavis92) [inline src/functions/scroll.ts]: Could we use the single parameter object pattern, like `v1` has?

If possible I'd like to keep this consistent, as it is better practice for maintaining functions as well as dev usability.

Comment (itdrake91) [reply src/functions/scroll.ts]: Good point. I'll try to make the arguments more similar to v1.

Comment (geoffdavis92) [reply src/functions/scroll.ts]: @itdrake91 thanks!

**Comment by geoffdavis92** [comment]
Thanks @itdrake91!

**Pull request merged**